e2e/paths/caseSubmitted_test.js

```diff
@@ -26,3 +31,13 @@ Scenario('Cannot submit a case unless consent is given', I => {
   I.click('Continue');
   I.seeInCurrentUrl('/submitApplication');
 });
+
+xScenario('Can submit a case and see date submitted', (I, caseViewPage, caseListPage, submitApplicationPage) => {
```

Comment:
could you add a comment as to why this test is being skipped for now and what needs to be done to un skip

Reply:
👍

Reply:
reason is more - we are awaiting the search functionality. When those changes are made we will search for a specific case and the paginiation stops being an issue

Reply:
I see no much point adding test that won't be executed

Reply:
I think it is good to have atleast one test